plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -692,6 +713,13 @@ fu_uefi_capsule_plugin_coldplug_device(FuPlugin *plugin, FuUefiCapsuleDevice *de
 					   FU_UEFI_CAPSULE_DEVICE_FLAG_COD_INDEXED_FILENAME);
 	}
 
+	/* if the bootloader doesn't know how to install capsules provided by fwupd,
+	 * enable fwupd-efi */
+	if (!fu_uefi_capsule_plugin_bootloader_supports_fwupd(ctx)) {
```

Comment:
that's still doing it 12 times tho. How about moving the check to `fu_uefi_capsule_plugin_coldplug()` (outside of the loop, and storing to a local `gboolean bootloader_supports_fwupd`) and then adding the flag there?

Reply:
Ah I missed the outer loop. Moved to a local flag in `fu_uefi_capsule_plugin_coldplug`.